plugins/experimental/sslheaders/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@
 #######################
 
 add_library(sslhdr STATIC expand.cc util.cc)
-target_link_libraries(sslhdr PRIVATE OpenSSL::SSL ts::tsutil)
```

Comment:
Should it also be removed on L24?

Reply:
Its fine to link in unit tests, this PR is to fix ODR violations in plugins

Reply:
It looks like it isn't needed, but maybe that can be addressed in a separate PR.